src/NUnitFramework/framework/Constraints/ConstraintExpression.cs

```diff
@@ -949,11 +949,27 @@ public SamePathOrUnderConstraint SamePathOrUnder(string expected)
         /// Returns a constraint that tests whether the actual value falls 
         /// within a specified range.
         /// </summary>
+        /// <remarks>from must be less than or equal to true</remarks> 
+        /// <param name="from">Inclusive beginning of the range. Must be less than or equal to to.</param>
+        /// <param name="to">Inclusive end of the range. Must be greater than or equal to from.</param>
         public RangeConstraint InRange(IComparable from, IComparable to)
```

Comment:
Comparer already defaults to `ComparisonAdapter.Default` down in RangeConstraint. But I think that's the NUnit-y equivalent of `Comparer<object>.Default`.

The optional parameter is a good point - although I have a vague memory that this signature was added to prevent users using the constraint to compare non-IComparables by default comparer? I don't see that that's strictly necessary though...maybe @rprouse or @CharliePoole who did the earlier reviews can remember the context better?